src/Sarif/Writers/ConsoleLogger.cs

```diff
@@ -2,17 +2,18 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Collections.Generic;
 using System.Globalization;
 using System.Linq;
 using System.Text;
 
 namespace Microsoft.CodeAnalysis.Sarif.Writers
 {
-    public class ConsoleLogger : IAnalysisLogger
+    public class ConsoleLogger : BaseLogger, IAnalysisLogger
```

Comment:
>ConsoleLogger [](start = 17, length = 13)

Note James: our console output should emit results in an ordered and deterministic way!

Reply:
Per our emails, created an issue here: [https://github.com/microsoft/sarif-sdk/issues/2250](https://github.com/microsoft/sarif-sdk/issues/2250)

---
In reply to: [559749330](https://github.com/microsoft/sarif-sdk/pull/2241#discussion_r559749330) [](ancestors = 559749330)